docs/plugins/integrations.asciidoc

```diff
@@ -29,13 +28,24 @@ make it easier to work with Elasticsearch.
   search (facets, etc), along with some Natural Language Processing features
   (ex.: More like this)
 
-
 [float]
 [[data-integrations]]
 === Data import/export and validation
 
-NOTE: Rivers were used to import data from external systems into
-Elasticsearch, but they are no longer supported in Elasticsearch 2.0.
+NOTE: Rivers were used to import data from external systems into Elasticsearch prior to the 2.0 release. Elasticsearch 
+releases 2.0 and later do not support rivers.
+
+[float]
+==== Supported by the community:
```

Comment:
Aw, that's fine! I'll go ahead and merge this.

On Thu, Sep 10, 2015 at 2:30 PM, David Pilato <notifications@github.com>
wrote:

> In docs/plugins/integrations.asciidoc
> <https://github.com/elastic/elasticsearch/pull/13446#discussion_r39216505>
> :
>
> >  [float]
> >  [[data-integrations]]
> >  === Data import/export and validation
> >
> > -NOTE: Rivers were used to import data from external systems into
> > -Elasticsearch, but they are no longer supported in Elasticsearch 2.0.
> > +NOTE: Rivers were used to import data from external systems into Elasticsearch prior to the 2.0 release. Elasticsearch
> > +releases 2.0 and later do not support rivers.
> > +
> > +[float]
> > +==== Supported by the community:
>
> @palecur <https://github.com/palecur> Ignore me! I'm sorry. I totally
> misread this section...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/elastic/elasticsearch/pull/13446/files#r39216505>.
>